apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -150,6 +151,47 @@ Page {
                 }
                 onClicked: pageStack.push("PermissionPage.qml")
             }
+
+            BrowserListItem {
+                //% "Save destination"
+                label: qsTrId("settings_browser-la-save_destination")
+                iconSource: "image://theme/icon-m-download"
+                value: {
+                    if (WebEngineSettings.useDownloadDir) {
+                        //% "Download dir"
+                        return qsTrId("sailfish_browser-me-download_dir")
```

Comment:
Again, this may be part of the fixed design, but why not just give the folder name here, and avoid the use of `secondaryText`?:

"Download to: /home/defaultuser/Downloads"
"Download to: Always ask"

Reply:
Again, please ignore more earlier comment.

It looks like the string used here should be either "Downloads" or the leafname of the folder. Personally I think the leafname would be clearer.